mlflow/store/file_store.py

```diff
@@ -353,3 +361,11 @@ def log_param(self, run_uuid, param):
         param_path = self._get_param_path(run.info.experiment_id, run_uuid, param.key)
         make_containing_dirs(param_path)
         write_to(param_path, "%s\n" % param.value)
+
+    def set_tag(self, run_uuid, tag):
+        _validate_run_id(run_uuid)
+        _validate_tag_name(tag.key)
+        run = self.get_run(run_uuid)
+        tag_path = self._get_tag_path(run.info.experiment_id, run_uuid, tag.key)
+        make_containing_dirs(tag_path)
```

Comment:
If a tag already exists on this run, what do we do? Update or overwrite? Case can be made for both use cases.

Reply:
Overwrite! This is tested in test_file_store.